misc/Makefile

```diff
@@ -1,12 +1,34 @@
 include ../Make.defs
 
-all clean:
+VERSION != grep 'Version:' ../tcb.spec | sed -e 's!Version:[ \t]*!!g'
+
+all: tcb.pc tcb.sysusers
+
+tcb.pc:
+	sed -e 's!@PREFIX@!$(PREFIX)!g' -e 's!@LIBDIR@!$(LIBDIR)!g' \
+		-e 's!@INCLUDEDIR@!$(INCLUDEDIR)!g' -e 's!@VERSION@!$(VERSION)!g' \
+		< tcb.pc.in > tcb.pc
```

Comment:
This way `tcb.pc` doesn't depend on `tcb.pc.in`, is this intended?

Reply:
Will fix on rebasing.

Reply:
Fixed.